### DISCUSSION TITLE
Is there a way to call functions?

### DISCUSSION THREAD

**Discussion by JakeTellestio** [srcnalt/OpenAI-Unity#120] [Q&A]
In the API you can call functions in the "Tools" section. But I don't see it implemented in the CreateChatCompletionRequest class. https://platform.openai.com/docs/api-reference/chat/create 
![image](https://github.com/srcnalt/OpenAI-Unity/assets/33091561/bc3badc4-4fc7-447f-b429-c203311c1f2b)

**Comment by rubickanov** [comment]
Hi there!

Better late than never, right?

Functions calling are not supported in this package. You might need to modify code. See #93
